.github/workflows/posix-deps-apt.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
+
+# Workaround missing libmpdec-dev on ubuntu 24.04:
+# https://launchpad.net/~ondrej/+archive/ubuntu/php
+# https://deb.sury.org/
+sudo add-apt-repository ppa:ondrej/php
```

Comment:
Can't we have a apt install libmpdec from this repository only so that we don't have multiple possible vendors? just in case. I don't know ehat this repo provides but I rather pull libs from main repos before pulling from other repos for libs that aren't in the main ones